capella_model_explorer/__main__.py

```diff
@@ -118,12 +109,13 @@ def run_local():
 
 
 def run_local_dev() -> None:
-    uvicorn = shutil.which("uvicorn")
-    if uvicorn is None:
+    if not (uvicorn := shutil.which("uvicorn")):
         raise SystemExit("`uvicorn` not found, did you run `pip install`?")
-    print("Running the application locally with full reload...")
+
+    logger.info("Running the application locally with full reload...")
     if not pathlib.Path(c.TEMPLATES_DIR).is_dir():
         raise SystemExit(f"Templates directory not found: {c.TEMPLATES_DIR}")
+
```

Comment:
My aim is to avoid unnecessary empty lines. I am not there yet and at the same time I wish that we avoid them.